maker-frontend/src/MakerApp.tsx

```diff
@@ -63,14 +63,23 @@ export default function App() {
     const [symbol, setSymbol] = useState("btcusd");
     let symbolDefaults = Defaults[symbol];
 
-    let source = useEventSource({ source: `/api/${symbol}/feed`, options: { withCredentials: true } });
+    let source = useEventSource({ source: `/api/feed`, options: { withCredentials: true } });
 
     let [leverages, setLeverages] = useState(["1", "2", "3"]);
 
     const cfdsOrUndefined = useLatestEvent<Cfd[]>(source, "cfds", intoCfd);
     let cfds = cfdsOrUndefined ? cfdsOrUndefined! : [];
-    const makerLongOrder = useLatestEvent<MakerOffer>(source, "long_offer");
-    const makerShortOrder = useLatestEvent<MakerOffer>(source, "short_offer");
+    const makerLongOrder = useLatestEvent<MakerOffer>(
+        source,
+        "long_offer",
+        undefined,
+        (data: any) => data && data.contract_symbol == "BtcUsd",
+    );
+    const makerShortOrder = useLatestEvent<MakerOffer>(
+        source,
+        "short_offer",
+        (data: any) => data && data.contract_symbol == "BtcUsd",
```

Comment:
❌ If we hard-code this to `BtcUsd` we will not filter the correct offers when we flip the switch. I think you need to use the variable `symbol` on line 63.

Reply:
It was hardcoded before so I did not change that as it wasn't part of the issue. Happy to change though.

Reply:
AFAIK it wasn't hard-coded: we had a dynamic source based on the value of `symbol`, which changed every time `setSymbol` was called.

The way I see it, with this changes when we change the symbol to `ETHUSD` we wills till display `BTCUSD`.

Reply:
you are right.. must have changed it myself and thought it has been like that before 🙈